include/zephyr/devicetree.h

```diff
@@ -2534,6 +2623,26 @@
 #define DT_INST_STRING_UPPER_TOKEN(inst, prop) \
 	DT_STRING_UPPER_TOKEN(DT_DRV_INST(inst), prop)
 
+/**
+ * @brief Get an element out of string-arrap property as a token.
+ * @param inst instance number.
```

Comment:
Are the periods at the end of each parameter new in the latest push? Maybe I just didn't notice them before. But please remove them for consistency with the other docstrings. Example: `@param inst instance number`, not `@param inst instance number.`